src/app/providers/externalMessagePort.provider/externalMessagePort.provider.stories.ts

```diff
@@ -89,12 +115,10 @@ const template = () => html`
       padding: 0;
     }
   </style>
-  <iframe
-    isncsci
-    src="https://brave-meadow-05543dc10.4.azurestaticapps.net/"
-  ></iframe>
+  <iframe isncsci src="http://localhost:44333/"></iframe>
```

Comment:
This change needs to be rolled back so the story continues to point at the dev site